source/globalCommands.py

```diff
@@ -1907,6 +1913,16 @@ def script_braille_routeTo(self, gesture):
 	script_braille_routeTo.__doc__ = _("Routes the cursor to or activates the object under this braille cell")
 	script_braille_routeTo.category=SCRCAT_BRAILLE
 
+	def script_braille_reportFormatting(self, gesture):
+		info = braille.handler.getTextInfoForWindowPos(gesture.routingIndex)
+		if info is None:
+			# Translators: Reported when trying to obtain formatting information (such as font name, indentation and so on) but there is no formatting information for the text under cursor.
+			ui.message(_("No formatting information"))
+			return
+		self._reportFormatting(info, False)

```

Comment:
I wonder whether reporting formatting info in a browsable message would make sense for routing with a braille display, e.g. after two or three presses?